src/main/java/chessmaster/game/Game.java

```diff
@@ -45,6 +46,10 @@ public void run() {
             try {
                 if (currentPlayer.isHuman()) {
                     command = getUserCommand();
+                    if (command.isShowMovesCommand()) {
```

Comment:
`ShowMoveCommand` objects should be executed during the `execute` method. 
The `execute` method in Command has the current `board` object so you can use that to call the `showAvailableMoves` method.